src/components/Medicine/MedicationAdministration/AdministrationTab.tsx

```diff
@@ -218,11 +220,17 @@ const MedicationRow: React.FC<MedicationRowProps> = ({
   );
 
   return (
-    <React.Fragment>
+    <React.Fragment key={medication.id}>
       <div
-        className={`p-4 border-t border-gray-200 ${isInactive ? "bg-gray-100" : ""}`}
+        className={cn("p-4 border-t border-gray-200", {
+          "opacity-40 bg-gray-200": isInactive,
```

Comment:
```suggestion
          isInactive && "opacity-40 bg-gray-200",
```

This technically works, but rewrite it so it is more readable